### PR TITLE
Add travis CI to DQC

### PR DESCRIPTION
**Pull request by jasonleinbach-wf**
#### Changes:

- Added travis CI support - this is enabled on my fork.  Once merged, we should be able to enable it on this repo.
- TODO: follow-up to get all of the test cases running.  This just runs unit tests and flake8.

Please review: @hefischer  @andrewperkins-wf

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1 +1 @@
--e git+ssh://git@github.com/arelle/arelle.git@master#egg=Arelle-dev
+-e git+https://git@github.com/arelle/arelle.git@master#egg=Arelle-dev
```
Comment (chrislococo-wf) [inline requirements.txt]: I can't wait to get to the point where arelle is pip-installable.

**Comment by chrislococo-wf** [comment]
+1

Where do we see the results?

**Comment by jasonleinbach-wf** [comment]
Currently it only runs on my fork: https://travis-ci.org/jasonleinbach-wf/dqc_us_rules

**Comment by jasonleinbach-wf** [comment]
image

**Comment by jasonleinbach-wf** [comment]
We will have to get someone with access to turn Travis on for the DQC repo.

**Comment by chrislococo-wf** [comment]
+1

**Comment by sagesmith-wf** [comment]
+1

**Comment by mindeerickson-wf** [comment]
Took a look at the Travic CI build: https://travis-ci.org/jasonleinbach-wf/dqc_us_rules

QA +1

**Comment by jasonleinbach-wf** [comment]
Reviewed and verified.  Merging.

**Pull request merged**